R/tidy-upkeep.R

```diff
@@ -140,7 +140,7 @@ upkeep_checklist <- function(year = NULL,
       todo("
         Update logo (https://github.com/rstudio/hex-stickers); \\
         run `use_tidy_logo()`"),
-      todo('Use `pak::pkg_install("org/pkg")` in README'),
+      todo('Use `pak::pak("org/pkg")` in README'),
```

Comment:
```suggestion
      todo('Prefer `pak::pak("org/pkg")` over `devtools::install_github("org/pkg")` in README'),
```

Let's be clear that this is about "to install the development version ...".